docs/essentials/exceptions/exceptions-hiearchy.md

```diff
@@ -0,0 +1,29 @@
+# Exception hierarchy
+
+Exceptions can be generated by an executing program or the common language runtime, or they can be thrown by an application or the runtime.
+
+[Exception](http://dotnet.github.io/api/System.Exception.html) is the base class for all exceptions in .NET Core. Most exceptions that derive directly from **Exception** add no functionality to the **Exception** class. For example, the [InvalidCastException](http://dotnet.github.io/api/System.InvalidCastException.html) class hierarchy is as follows:
```

Comment:
> For example, the [InvalidCastException](http://dotnet.github.io/api/System.InvalidCastException.html) class hierarchy is as follows:

This should be the second sentence. It seems like it is a "for example" for the second sentence, but it is not.